diff --git a/node_modules/chakra-datetime-picker/dist/esm/index.js b/node_modules/chakra-datetime-picker/dist/esm/index.js
index 4c2bafe..a191aff 100644
--- a/node_modules/chakra-datetime-picker/dist/esm/index.js
+++ b/node_modules/chakra-datetime-picker/dist/esm/index.js
@@ -3,6 +3,7 @@ import $26Zo0$dayjs, {isDayjs as $26Zo0$isDayjs} from "dayjs";
 import $26Zo0$dayjspluginisBetween from "dayjs/plugin/isBetween";
 import $26Zo0$dayjspluginisSameOrAfter from "dayjs/plugin/isSameOrAfter";
 import $26Zo0$dayjspluginisSameOrBefore from "dayjs/plugin/isSameOrBefore";
+import "dayjs/locale/ja";
 import {useRef as $26Zo0$useRef, useEffect as $26Zo0$useEffect, useState as $26Zo0$useState, useMemo as $26Zo0$useMemo, useCallback as $26Zo0$useCallback, forwardRef as $26Zo0$forwardRef, useImperativeHandle as $26Zo0$useImperativeHandle} from "react";
 import {FaAngleLeft as $26Zo0$FaAngleLeft, FaAngleDoubleLeft as $26Zo0$FaAngleDoubleLeft, FaArrowLeft as $26Zo0$FaArrowLeft, FaAngleRight as $26Zo0$FaAngleRight, FaAngleDoubleRight as $26Zo0$FaAngleDoubleRight, FaArrowRight as $26Zo0$FaArrowRight, FaRegCalendar as $26Zo0$FaRegCalendar} from "react-icons/fa";
 import {Box as $26Zo0$Box, Flex as $26Zo0$Flex, IconButton as $26Zo0$IconButton, Button as $26Zo0$Button, Portal as $26Zo0$Portal, Popover as $26Zo0$Popover, InputGroup as $26Zo0$InputGroup, Input as $26Zo0$Input, InputRightElement as $26Zo0$InputRightElement, PopoverTrigger as $26Zo0$PopoverTrigger, PopoverContent as $26Zo0$PopoverContent, PopoverBody as $26Zo0$PopoverBody} from "@chakra-ui/react";
@@ -18,6 +19,8 @@ import {Box as $26Zo0$Box, Flex as $26Zo0$Flex, IconButton as $26Zo0$IconButton,
 $26Zo0$dayjs.extend($26Zo0$dayjspluginisSameOrBefore);
 $26Zo0$dayjs.extend($26Zo0$dayjspluginisSameOrAfter);
 $26Zo0$dayjs.extend($26Zo0$dayjspluginisBetween);
+$26Zo0$dayjs.locale("ja");
+
 const $05beb4bfea48a2e2$var$_disabled = {
     backgroundColor: "gray.50",
     color: "gray.200",
@@ -54,7 +57,7 @@ const $05beb4bfea48a2e2$var$defaultProps = {
         return;
     },
     cancelText: "Cancel",
-    currentLangKey: "zh"
+    currentLangKey: "ja"
 };
 const $05beb4bfea48a2e2$var$componentSize = {
     md: {
@@ -195,6 +198,9 @@ const $05beb4bfea48a2e2$export$5109c6dd95d8fb00 = ({ picker: picker , defaultVal
             case "zh_cn":
                 now.locale("zh-cn");
                 break;
+            case "ja":
+                now.locale("ja");
+                break;
             default:
                 now.locale("en");
                 break;
@@ -429,7 +435,7 @@ const $05beb4bfea48a2e2$export$5109c6dd95d8fb00 = ({ picker: picker , defaultVal
                                 color: `${colorScheme}.700`,
                                 display: "flex",
                                 children: [
-                                    /*#__PURE__*/ $26Zo0$jsx($26Zo0$Box, {
+                                    picker !== "anniversary" && /*#__PURE__*/ $26Zo0$jsx($26Zo0$Box, {
                                         padding: $05beb4bfea48a2e2$var$componentSize[size].monthAndYear.padding,
                                         cursor: "pointer",
                                         transition: "color 0.25s",
@@ -437,11 +443,11 @@ const $05beb4bfea48a2e2$export$5109c6dd95d8fb00 = ({ picker: picker , defaultVal
                                             color: `${colorScheme}.200`
                                         },
                                         onClick: ()=>{
-                                            setView("month");
+                                            setView("year");
                                         },
-                                        children: view === "datetime" && currMonth.format("MMM")
+                                        children: view === "year" ? `${recentTenYear.format("YYYY") + "年"}-${recentTenYear.add(9, "year").format("YYYY") + "年"}` : currMonth.format("YYYY") + "年"
                                     }),
-                                    picker !== "anniversary" && /*#__PURE__*/ $26Zo0$jsx($26Zo0$Box, {
+                                    /*#__PURE__*/ $26Zo0$jsx($26Zo0$Box, {
                                         padding: $05beb4bfea48a2e2$var$componentSize[size].monthAndYear.padding,
                                         cursor: "pointer",
                                         transition: "color 0.25s",
@@ -449,10 +455,10 @@ const $05beb4bfea48a2e2$export$5109c6dd95d8fb00 = ({ picker: picker , defaultVal
                                             color: `${colorScheme}.200`
                                         },
                                         onClick: ()=>{
-                                            setView("year");
+                                            setView("month");
                                         },
-                                        children: view === "year" ? `${recentTenYear.format("YYYY")}-${recentTenYear.add(9, "year").format("YYYY")}` : currMonth.format("YYYY")
-                                    })
+                                        children: view === "datetime" && currMonth.format("MMM")
+                                    }),
                                 ]
                             }),
                             !(picker === "anniversary" && view === "month") && /*#__PURE__*/ $26Zo0$jsx($26Zo0$IconButton, {
@@ -552,7 +558,7 @@ const $05beb4bfea48a2e2$export$5109c6dd95d8fb00 = ({ picker: picker , defaultVal
                                     ...isDisabled && {
                                         _hover: {}
                                     },
-                                    children: year.format("YYYY")
+                                    children: year.format("YYYY") + "年"
                                 }, i);
                             })
                         })
diff --git a/node_modules/chakra-datetime-picker/dist/index.d.ts b/node_modules/chakra-datetime-picker/dist/index.d.ts
index 1e09b7b..2300597 100644
--- a/node_modules/chakra-datetime-picker/dist/index.d.ts
+++ b/node_modules/chakra-datetime-picker/dist/index.d.ts
@@ -5,7 +5,7 @@ interface DatePickerProps {
     cancelButtonProps?: ButtonProps;
     cancelText?: string;
     colorScheme?: string;
-    currentLangKey?: "en" | "zh" | "zh_cn";
+    currentLangKey?: "en" | "zh" | "zh_cn" | "ja";
     defaultValue?: string | number | Dayjs;
     disableTimestampAfter?: number | false;
     disableTimestampBefore?: number | false;
